backend/tests/unit/records/marshmallow/experiments/test_es.py

```diff
@@ -45,44 +45,19 @@ def test_experiment_serializer_populates_experiment_suggest():
         "name_variants": ["NAME_V1", "NAME_V2", "NAME_V3"],
     }
 
-    expected_experiment_suggest = {
-        "input": [
-            "ACC",
-            "COLLABORATION",
-            "EXP SHORT NAME",
-            "Experiment value",
-            "INST_VALUE",
-            "LEGACY-NAME",
-            "{Long Name}",
-            "NAME_V1",
-            "NAME_V2",
-            "NAME_V3",
-        ]
-    }
+    expected_experiment_suggest = [
+        {"input": "ACC", "weight": 1},
+        {"input": "COLLABORATION", "weight": 1},
+        {"input": "EXP SHORT NAME", "weight": 1},
+        {"input": "Experiment value", "weight": 1},
+        {"input": "INST_VALUE", "weight": 1},
+        {"input": "{Long Name}", "weight": 1},
+        {"input": "NAME_V1", "weight": 1},
+        {"input": "NAME_V2", "weight": 1},
+        {"input": "NAME_V3", "weight": 1},
+        {"input": "LEGACY-NAME", "weight": 5},
```

Comment:
don't we need to change something on UI part?

Reply:
as far as I can see from the docs, input value that matched always comes in `options[].text`, weight is just used for matching.